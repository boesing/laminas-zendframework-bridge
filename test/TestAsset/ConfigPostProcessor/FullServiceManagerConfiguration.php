<?php

return [
    'service_manager' => [
        'abstract_factories' => [
            'Zend\ServiceManager\AbstractFactory\ConfigAbstractFactory',
        ],
        'aliases' => [
            'Zend\Cache\Storage\StorageInterface' => 'Zend\Cache\Storage\Adapter\Redis',
        ],
        'delegators' => [
            'Zend\Cache\Storage\Adapter\Redis' => [
                'Zend\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
        'factories' => [
            'Zend\Form\Factory' => 'Some\Vendor\Zend\Form\ZendFormFactory',
            'MyService' => 'Zend\ServiceManager\Factory\InvokableFactory',
        ],
        'initializers' => [
            // Just for testing purposes, this initializer does not exist
            'Zend\Form\FactoryInitializer',
        ],
        'invokables' => [
            'Zend\Expressive\Router\RouterInterface' => 'MyService',
        ],
        'lazy_services' => [
            'class_map' => [
                'Zend\Cache\Storage\Adapter\Redis' => 'Zend\Cache\Storage\Adapter\Redis',
            ],
        ],
        'services' => [
            // NOTE: this is an invalid configuration, you have to provide service instances in this array.
            // For testing purposes, we have to change this to a string we can use to verify that the value
            // is not touched at all. Using instances would lead to
            'Zend\Cache\Storage\Adapter\Redis' => 'Zend\Cache\Storage\Adapter\Redis',
        ],
        'shared' => [
            'Zend\Form\Factory' => false,
        ],
    ],
];
