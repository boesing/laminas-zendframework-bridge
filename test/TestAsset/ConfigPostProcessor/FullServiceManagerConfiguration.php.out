<?php

return [
    'service_manager' => [
        'abstract_factories' => [
            'Laminas\ServiceManager\AbstractFactory\ConfigAbstractFactory',
        ],
        'aliases' => [
            'Zend\Cache\Storage\StorageInterface' => 'Laminas\Cache\Storage\StorageInterface',
            'Laminas\Cache\Storage\StorageInterface' => 'Laminas\Cache\Storage\Adapter\Redis',
            'Zend\Expressive\Router\RouterInterface' => 'Mezzio\Router\RouterInterface',
            'Zend\Form\Factory' => 'Laminas\Form\Factory',

        ],
        'delegators' => [
            'Laminas\Cache\Storage\Adapter\Redis' => [
                'Laminas\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
        'factories' => [
            'MyService' => 'Laminas\ServiceManager\Factory\InvokableFactory',
            'Laminas\Form\Factory' => 'Some\Vendor\Zend\Form\ZendFormFactory',
        ],
        'initializers' => [
            // Just for testing purposes, this initializer does not exist
            'Laminas\Form\FactoryInitializer',
        ],
        'invokables' => [
            'Mezzio\Router\RouterInterface' => 'MyService',
        ],
        'lazy_services' => [
            'class_map' => [
                'Laminas\Cache\Storage\Adapter\Redis' => 'Laminas\Cache\Storage\Adapter\Redis',
            ],
        ],
        'services' => [
            'Laminas\Cache\Storage\Adapter\Redis' => 'Zend\Cache\Storage\Adapter\Redis',
        ],
        'shared' => [
            'Laminas\Form\Factory' => false,
        ],
    ],
];
